ee/modules/502-delivery/templates/argocd/server/argocd-server-serviceaccount.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app.kubernetes.io/component: server
```

Comment:
For some resources, Deckhouse label helper is used, but not everywhere (can lead to matrix tests errors).
`  {{- include "helm_lib_module_labels" (list . (dict "app" "argocd")) | nindent 2 }}`

Reply:
done